confluence/client.py

```diff
@@ -463,6 +463,27 @@ def get_attachments(self, content_id, filename=None, media_type=None, expand=Non
                                        params=params,
                                        expand=expand)
 
+    def download_attachment(self, attachment):
```

Comment:
I'll need an integration test for this in line with the others, should be fairly simple to add but let me know if it's not

Reply:
The build failed here: https://travis-ci.org/DaveTCode/confluence-python-lib.

There's some issues that probably aren't yours but your specific new test doesn't pass. I'll take a look at the others tomorrow but could you ensure your new test passes locally?

Reply:
The build also fails on mypy checks on line 467 in this file, something else to resolve locally

Reply:
Oops, sorry about that, code wasn't tested properly.  Latest commits passes the integration testing locally and passes mypy

Reply:
And now the tests as well as integration_tests